.github/workflows/comment-stalled.yml

```diff
@@ -0,0 +1,22 @@
+name: Comment on issues and PRs when labelled stalled
+on:
+  issues:
+    types: [labeled]
+  pull_request:
+      types: [labeled]
+  pull_request_target:
+    types: [labeled]
```

Comment:
This won't work for existing PRs, but maybe that's ok.

Reply:
I fumbled around quite a bit myself before discovering that as well... You know ways to circumvent that by any chance?

Reply:
Not yet :/